src/topology/algebra/module/finite_dimension.lean

```diff
@@ -0,0 +1,356 @@
+/-
+Copyright (c) 2022 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel, Anatole Dedecker
+-/
+import analysis.locally_convex.balanced_core_hull
+
+/-!
+# Finite dimensional topological vector spaces over complete fields
```

Comment:
Can you add a comment somewhere that reminds the reader how "topological vector space" is spelt in lean?